service-loadbalancer/service_loadbalancer.go

```diff
@@ -388,8 +390,9 @@ func (lbc *loadBalancerController) getServices() (httpSvc []service, tcpSvc []se
 				continue
 			}
 			newSvc := service{
-				Name: getServiceNameForLBRule(&s, servicePort.Port),
-				Ep:   ep,
+				Name:        getServiceNameForLBRule(&s, servicePort.Port),
+				Ep:          ep,
+				BackendPort: servicePort.Port,
```

Comment:
You're going to use this for health checks on the endpoint right? don't you want the endpoint port? I mean, you can have an nginx service with port 80, and targetport 8080 going to an endpoint running on port 8080.